tests/regression/roles/sonic_port_group/defaults/main.yml

```
@@ -37,7 +37,7 @@ tests:
     description: Set all port group to default speeds
```

It is just one port group, now, so please change this line to something like:

```suggestion
    description: Set a port group to the default speed.
```

---

Done. Thanks!

---

Thank you.

Resolved.